libs/dkg/include/dkg/dkg_complaints_manager.hpp

```diff
@@ -59,8 +59,8 @@ class ComplaintsManager
 
   void ResetCabinet(uint32_t cabinet_size);
   void Count(MuddleAddress const &address);
-  void Add(std::shared_ptr<ComplaintsMessage> const &msg_ptr, MuddleAddress const &from_id,
-           uint32_t from_index, MuddleAddress node_address);
+  void Add(ComplaintsMessage const &msg, MuddleAddress const &from_id, uint32_t from_index,
```

Comment:
This is different now, but in the previous code it was a ref to a shared pointer - was this done so it could be null?

Reply:
It should not be null and that is why it has been changed. The message should not get to this stage if it is null but I have changed to reference to be safe.